events/events_network.c

```diff
@@ -1,4 +1,5 @@
 #include <sys/select.h>
+#include <sys/time.h>
```

Comment:
Not needed; POSIX says `The <sys/select.h> header shall define the timeval structure`

Reply:
Oops.  I always forget about that one.  I did double-check that it's not in `<time.h>`.